doc/devdocs/types.rst

```diff
@@ -319,9 +319,11 @@ the type, which is an object of type :obj:`TypeName`:
      primary: Array{T,N} <: DenseArray{T,N}
      cache: SimpleVector
        ...
+
      linearcache: SimpleVector
        ...
-     uid: Int64 47
+
+     uid: Int64 -7900426068641098781
```

Comment:
Oh, I copied this weird number from the error trace I got (albeit consistently). I think that's just a random integer (`rand(Int`). Also, I changed the number and ran a few times, and it passed, so I don't think this matter. Would you prefer a less weird number up here? :)